packages/terra-section-header/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "terra-section-header",
+  "main": "lib/SectionHeader.js",
+  "version": "0.0.0",
+  "description": "Interactive header component that provides a status indicator.",
```

Comment:
update the comment to be something like "Section Header component that contains text and can be optionally interacted with."

Reply:
Done afb88082210e62561c5c043ebd9a197b11fa785f